scripts/add-list-items.js

```diff
@@ -0,0 +1,17 @@
+export default function addListItem(event, input, list) {
+  // Prevent button from scrolling up
+  event.preventDefault();
+
+  // Remove empty space from input
+  const inputValue = input.value.trim();
```

Comment:
`inputValue` should be the parameter, not the entire input field. that way, your `addListItem` function is not dependent on a specific input field and you can easily refactor it later. 
you can always read the `input.value` before invoking the `addListItem` function and separate the concerns.